### ISSUE TITLE
Get ringer mode

### ISSUE THREAD

**Issue by squidink7** [k3yb0ardn1nja/nativescript-ringer#1]
I'm making an application which requires the detection of the current ringer mode before setting it, would that be possible to implement in this library? Thanks!

**Comment by digitaldoggo** [comment]
I created a branch with this feature implemented. I don't have the CLI installed at the moment, I will try to get the code tested and get the package updated. In the meantime, you could side-load the feature branch to test the added function with the following.

Clone the repo:
```sh
git clone https://github.com/digitaldoggo/nativescript-ringer
```

Change to this repo directory:
```sh
cd nativescript-ringer
```

Switch to the feature branch:
```sh
git checkout feature/get-mode
```

Package the plugin:
```sh
npm pack
```

From your project's directory, install the plugin package we created:
```sh
npm i /path/to/nativescript-ringer-0.2.0.tgz
```

Let me know how it works if you give it a try.

**Comment by digitaldoggo** [comment]
I published v1.0.0, which adds the method and updates to support NativeScript 8. The usage changed slightly, as seen in the readme. Let me know if you have any trouble with it.
